core/camel-api/src/main/java/org/apache/camel/Route.java

```diff
@@ -373,4 +373,9 @@
      */
     void addErrorHandlerFactoryReference(ErrorHandlerFactory source, ErrorHandlerFactory target);
 
+    /**
+     * Adds a resume strategy to the route
+     */
+    void setResumeStrategy(ResumeStrategy resumeStrategy);
```

Comment:
add or set ?

Reply:
Good catch. The phrasing should be "set".